docs/UserGuide.md

```diff
@@ -226,10 +237,10 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+FinCliq data are saved automatically as a JSON file `[JAR file location]/data/fincliq.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
This means we have to change the name of the data file from `addressbook.json` to `fincliq.json`.